components/print/hplip/patches/16_libImageProcessor.arch.patch

```diff
@@ -0,0 +1,101 @@
+
+Make createPPD.sh choose the architecture for libImageProcessor.so based
+on a compiled binary, not based on the kernel architecture.
+
+Fix Makefile.am and Makefile.in so that it installs the i386 version of
```

Comment:
The fix seems to be overcomplicated. Can we just pass  its as compile-time variable and do  something like COMPONENT_BUILD_ENV+="ARCH=$(BITS)".